barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy.test.cpp

```diff
@@ -571,10 +571,10 @@ TYPED_TEST(ProtogalaxyTests, CombineAlphas)
     TestFixture::test_compute_and_extend_alphas();
 }
 
-TYPED_TEST(ProtogalaxyTests, ProtogalaxyInhomogeneous)
-{
-    TestFixture::test_protogalaxy_inhomogeneous();
-}
+// TYPED_TEST(ProtogalaxyTests, ProtogalaxyInhomogeneous)
```

Comment:
Again, folding without structured trace now fails since polys dont agree in shape across instances. This was never a use case anyway.